src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,32 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.health = health || 100;
```

Comment:
```suggestion
  constructor(name, health = 100) {
    this.health = health;
```
You can just set the default value